web/modules/custom/bnf/bnf_client/src/Plugin/QueueWorker/SubscriptionNewContent.php

```diff
@@ -98,6 +98,11 @@ public function processItem($data): void {
     );
 
     foreach ($newContent['uuids'] as $uuid) {
+      $this->nodeQueue->createItem([
+        'uuid' => $uuid,
+        'categories' => $subscription->getCategories(),
+        'tags' => $subscription->getTags(),
+      ]);
       $this->nodeQueue->createItem(['uuid' => $uuid]);
```

Comment:
Huh? Double create?

Reply:
Mistake, after a faulty rebase. Thank you!